tests/my_house_tests.py

```diff
@@ -0,0 +1,162 @@
+import pytest
+from house.constants import (
+    MINE_ADD_EXPENSE_ROUTE,
+    MINE_EXPENSES_TABLE_ROUTE,
+    MINE_HOUSE_TABLE_ROUTE,
+    MINE_PAGE_ROUTE,
+    MINE_SIDEBAR_ROUTE,
+    MINE_EDIT_EXPENSE_ROUTE,
+)
+from factories.expense import ExpenseFactory
+from factories.house import HouseFactory
+from expenses.models import Expenses
+from django.template.loader import get_template
+from tests.const import (
+    EXPENSE_FORM_DATA,
+    EXPENSE_BAD_FORM_DATA_DESCRIPTION,
+    EXPENSE_BAD_FORM_DATA_AMOUNT,
+    EXPENSE_DESCRIPTION_BEFORE,
+    EXPENSE_DESCRIPTION_AFTER,
+    EXPENSE_AMOUNT,
+    EXPENSE_DATE,
+    EXPENSE_CATEGORY,
+)
+from house.forms import ExpenseForm
+
+
+@pytest.fixture
+def generate_expense_form():
+    form = ExpenseForm(EXPENSE_FORM_DATA)
+    return form
+
+
+@pytest.fixture
+def generate_bad_description_expense_form():
+    form = ExpenseForm(EXPENSE_BAD_FORM_DATA_DESCRIPTION)
+    return form
+
+
+@pytest.fixture
+def generate_bad_amount_expense_form():
+    form = ExpenseForm(EXPENSE_BAD_FORM_DATA_AMOUNT)
+    return form
+
+
+@pytest.mark.django_db
+class TestExpenseForm:
+    def test_expense_form(self, generate_expense_form):
+        form = generate_expense_form
+        assert form.is_valid()
+
+    def test_bad_form_amount(self, generate_bad_amount_expense_form):
+        form = generate_bad_amount_expense_form
+        assert not form.is_valid()
+
+    def test_bad_form_description(self, generate_bad_description_expense_form):
+        form = generate_bad_description_expense_form
+        assert not form.is_valid()
+
+
+@pytest.mark.django_db
+class TestMyHouseViewsAndTemplates:
+    def test_get_house_view_function(self, client, new_user):
+        client.force_login(new_user)
```

Comment:
Genereric err in tests
Look at the UI slides, should be done in @fixture

Reply:
fixed